### PR TITLE
Bump PHPCS version, removed sniffs merged to upstream

### PR DESCRIPTION
**Pull request by grongor**
Tested against our codebase, no errors ...

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,4 +1,4 @@
-Cdn77 Coding Standard [![Build Status](https://travis-ci.org/cdn77/coding-standard.svg)](https://travis-ci.org/cdn77/coding-standard)
+Cdn77 Coding Standard [![Build Status](https://travis-ci.org/cdn77/coding-standard.svg?branch=master)](https://travis-ci.org/cdn77/coding-standard)
```
Comment (simPod) [inline README.md]: 👍 noticed this thing recently, very weird from travis

Comment (grongor) [reply README.md]: yep, doesn't make sense really

File: composer.json [hunk 2]
```diff
@@ -17,13 +17,13 @@
...
         "doctrine/coding-standard": "^6.0",
-        "squizlabs/php_codesniffer": "^3.4.0"
+        "squizlabs/php_codesniffer": "^3.5"
     },
     "require-dev": {
         "jakub-onderka/php-parallel-lint": "^1.0",
-        "phpstan/phpstan": "^0.11.7",
-        "phpstan/phpstan-phpunit": "^0.11.2",
-        "phpstan/phpstan-strict-rules": "^0.11.0",
+        "phpstan/phpstan": "^0.11",
```
Comment (simPod) [inline composer.json]: we should probably lock to patch version, as phpstan's maintainer doesn't consider patch version to be patch version. When new "patch" version is released and new violation is found it breaks the build.

Comment (grongor) [reply composer.json]: oh... ok...sucks

Comment (simPod) [reply composer.json]: There was [ackward discussion](https://github.com/phpstan/phpstan/issues/2163) about it.

**Pull request merged**